src/api-endpoints.ts

```diff
@@ -62,12 +60,6 @@ type SelectColor =
   | "pink"
   | "red"
 
-type SelectPropertyResponse = {
```

Comment:
hm probably not related to your change, but it's not good that this got removed 🤔

Reply:
Yea I just ran `notion build-public-api-js-sdk --path=../notion-sdk-js/src/` which introduced a lot of other changes. I'm wondering if I should manually remove the changes I didn't introduce

Reply:
<img width="1282" alt="image" src="https://github.com/makenotion/notion-sdk-js/assets/157412996/8aa4a3f9-bec7-4eae-b6f7-6f16c251c761">